drv/update-api/hiffy-rot-flash.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/zsh
+
+# Load an image using the humility hiffy interface for the lpc55-update-server.
+#
+# Usage Example:
+# ARCHIVE=hubris/target/rot-carrier/dist/a/build-rot-carrier.zip rot-update-tools/hiffy-rot-flash.sh rot-carrier-v2/b/final.bin ImageB
```

Comment:
I think this should work on the SP as well, might be worth testing just so we have this as another reference

Reply:
It does appear to work on the SP as well, although it's extremely slow.